test/spec/geo/ui/attribution.spec.js

```diff
@@ -0,0 +1,119 @@
+var $ = require('jquery');
```

Comment:
Instead of relying so much on the internals (eg: calling `this.view._showAttributions();`) to test the behaviour of the attribution-view, I'd try to test it from the user's POV (i.e: simulating clicks and other events, and ensuring the view responds to this events by checking the DOM is updated accordingly). Here's some pseudocode:

```
it('should render the attributions when button is clicked', function () {
  // click .js-button
  // assert that elements are visible
})

describe('when the attributions are expanded', function () {

  beforeEach(function () {
    // click .js-button to expand attributions
  });
  
  it('should hide the button', function() {
   // assert that button is hidden
  });

  it('should collapse the attributions when ESC is pressed', function () {
    // simulate a ESCAPE keystroke
    // assert that elements are NOT visible
  });

  it('should collapse the attributions when user clicks on the document', function () {
    // trigger a click event on the document
    // assert that elements are NOT visible
  });

});

describe('when the attributions are collapsed', function () {
 
  beforeEach(function(){
    // click .js-button to expand attributions
    // simulate a ESCAPE keystroke to collapse the attributions
  });

  it('should show the button', function() {
    // assert that the button is visible
  });

  it('should NOT render the attributions', function () {
    // assert that elements are NOT visible
  });

  it('should not respond to ESC', function () {
    // simulate a ESCAPE keystroke
    // assert that elements are NOT visible
  });

  it('should not respond to clicks on the document', function () {
    // trigger a click event on the document
    // assert that elements are NOT visible
  });
});
```